pfcli/main.py

```diff
@@ -102,6 +109,24 @@ def login(
     else:
         _handle_login_response(r, False)
 
+    # Save user's organiztion context
+    project_client: ProjectClientService = build_client(ServiceType.PROJECT)
+    user_group_client: UserGroupClientService = build_client(ServiceType.USER_GROUP)
+    orgs = user_group_client.get_group_info()
+    org_id = orgs[0]["id"]  # TODO (ym): We assume single-org
```

Comment:
How about moving
```
org_id = orgs[0]["id"]  # TODO (ym): We assume single-org
```
into `get_group_info`? 
`get_group_info` just returns a dictionary of group information